packages/nocodb/src/models/LongTextColumn.ts

```
@@ -0,0 +1,113 @@
+      ));
+    if (!column) {
+      column = await ncMeta.metaGet2(
+        context.workspace_id,
+        context.base_id,
+        MetaTable.COL_LONG_TEXT,
+        { fk_column_id: columnId },
+      );
+      await NocoCache.set(`${CacheScope.COL_LONG_TEXT}:${columnId}`, column);
+    }
+
+    return column ? this.castType(column) : null;
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)